src/core/urdb.jl

```diff
@@ -290,12 +293,10 @@ function parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM
                     else
                         tier_use = tier
                     end
-                    if non_kwh_units
-                        rate = rate_average
-                    else
-                        rate = get(d["energyratestructure"][period][tier_use], "rate", 0)
-                    end
-                    total_rate = rate + get(d["energyratestructure"][period][tier_use], "adj", 0)
+                    total_rate = non_kwh_units ? 
```

Comment:
@hdunham I don't understand this change. When would `total_rate = non_kwh_units ? ` evaluate to true?